modules/web/src/app/core/components/sidenav/style.scss

```diff
@@ -70,3 +70,19 @@ $sidenav-item-margin: 10px;
     margin-right: 0;
   }
 }
+
+.sub-menu-item {
+  height: 30px;
+  margin: 0 0 10px 20px;
+  padding: 5px 35px;
+}
+
+.menu-title {
+  border-bottom: 2px solid gray;
+}
+
+.collapsed-sub-menu-item {
+  height: 25px;
+  margin: 0 0 10px 10px;
+  padding: 5px 10px;
+}
```

Comment:
```suggestion
.collapsed-sub-menu-item {
    height: 30px;
    padding: 5px 5px 5px 35px;
    position: relative;
}

.collapsed-sub-menu-item.active:after {
    position: absolute;
    left: 20px;
    top: 20%;
    height: 60%;
    width: 3px;
    content: '';
    background-color: #0072b0; // this can be moved to theme.scss
}
```